src/scss/style.scss

```diff
@@ -1,70 +1,96 @@
-$headerBgColor: slateblue;
-$bodyBgColor: lightgrey;
-
-h2{
-  font-size: 30px;
-  margin: 10px;;
-}
-
-
-body > * {
-  // border: 1px dotted gray;
-  // margin: 10px;
-  // padding: 5px;
+html{
+  width: 100%;
+  height: 100vh;
+  background-image: linear-gradient(127deg, #360033, #2d1b46 20%, #2c1d48 22%, #0b8793);
 }
-
-body {
-  background: $bodyBgColor;
-  font-family: Arial, Helvetica, sans-serif;
+body{
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
 }
-
-header {
+form{
   display: flex;
-  flex-direction: row;
-  align-items: baseline;
-  justify-content: space-between;
-  background: $headerBgColor;
+  flex-direction: column;
+  max-width:388px;
+  margin:auto;
+  background-color: #3DC2C5;
+  box-shadow: 10px 10px #112638;
 }
-
-ul {
-  display: inline-block;
+h1{
+  font-family: 'Libre Baskerville', serif;
+  font-size: 36px;
+  font-weight: bold;
+  color: #ffffff;
+  margin-bottom: -3%;
```

Comment:
set text margins and paddings in px instead of % - don't make them depend on the parent width